src/plugin.scss

```diff
@@ -1,9 +1,9 @@
 /* The default color for the playlist menu background, almost black */
-@background-color: #1a1a1a;
+$background-color: #1a1a1a;
```

Comment:
Unfortunately, this file had to be migrated to Sass. That said, I checked the rendered output against the current `latest` version's rendered output and they were identical.